src/views/SelectProduct.vue

```diff
@@ -112,42 +112,48 @@
         <main class="main">
           <section class="sort">
             <ion-item lines="none">
-              <h2>{{ $t("Results") }}:</h2>
+              <h2>{{ $t("Results") }}: {{ products.total.virtual }} {{ $t("virtual, ") }} {{ products.total.variant }} {{ $t("variants") }}</h2>
             </ion-item>
 
             <section class="search">
-              <ion-searchbar :placeholder="$t('Search products')" />
+              <ion-searchbar :placeholder="$t('Search products')" v-model="queryString" @keyup.enter="searchProducts($event)" />
             </section>  
           </section>
 
           <hr />
-
-          <section class="section-header">
-            <div class="primary-info">
-              <ion-item lines="none">
-                <ion-label>
-                  Parent Product
-                  <p>5 {{ $t("variants") }}</p>
-                </ion-label>
-              </ion-item>
-            </div>
-
+          <div v-for="product in products.list" :key="product">
+            <section class="section-header">
+              <div class="primary-info">
+                <ion-item lines="none">
+                  <ion-label>
+                    {{ product.productName }}
+                    <p>{{ product.variants.length}} {{ $t("variants") }}</p>
+                  </ion-label>
+                </ion-item>
+              </div>
+  
             <div class="tags"></div>
-          </section>
-
-          <section class="section-grid">
-            <ion-card>
-              <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
-              <ion-item lines="none">
-                <ion-label>
-                  SKU
-                  <p>Color: Blue</p>
-                  <p>Size: XL</p>
-                </ion-label>
-              </ion-item>
-            </ion-card>
-          </section>
-          <hr />
+            </section>
+  
+            <section class="section-grid">
+              <div v-for="variant in product.variants" :key="variant">
+                <ion-card>
+                  <Image :src="variant.mainImageUrl" />
+                  <ion-item lines="none">
+                    <ion-label>
+                      {{ variant.productName }}
+                      <p>{{ $t("Color") }}: {{ $filters.getFeature(variant.featureHierarchy, '1/COLOR/') }}</p>
```

Comment:
I think we should extract it while getting products

Reply:
Updated code to extract features related to each variant at the time of API call and directly use it on UI.